script.js

```diff
@@ -0,0 +1,172 @@
+      { NAME: "Whole cloves", AMOUNT: "5" },
+      { NAME: "Cinnamon sticks", AMOUNT: "2" },
+      { NAME: "Spice", AMOUNT: undefined },
+      { NAME: "Bottle of red wine", AMOUNT: "1" },
+      { NAME: "Raisins", AMOUNT: "100 g" },
+      { NAME: "Slipped Almonds", AMOUNT: "50 g" },
+    ],
+    description: "Mix everything, heat it, and you are good to go!",
+  },
+];
+
+let recipeCounter = 1;
+let ingredientNumber = 4;
+
+//*******************  show recipes in the page **********************/
+
+function addRecipeToPage(recipeObject) {
+  const recipePart = document.createElement("div");
+  recipePart.classList.add("Container");
+  const recipesContainer = document.getElementById("recipesContainer");
+  recipesContainer.appendChild(recipePart);
+
+  //*************** structure of show of recipes
+
+  const foodImageContainer = document.createElement("div");
+  foodImageContainer.classList.add("foodImageDiv");
+  recipePart.appendChild(foodImageContainer);
+  const foodImage = document.createElement("img");
+  foodImage.src = recipeObject.picture_url;
+  foodImageContainer.appendChild(foodImage);
+
+  //***************
+
+  const descriptionContainer = document.createElement("div");
+  descriptionContainer.classList.add("recipeContent");
+  recipePart.appendChild(descriptionContainer);
+
+  const foodName = document.createElement("h2");
+  foodName.classList.add("mainTitles");
+  foodName.innerText = recipeObject.title;
+  descriptionContainer.appendChild(foodName);
+
+  const ingredientsTitle = document.createElement("p");
+  ingredientsTitle.innerText = "Ingradients:";
+  ingredientsTitle.style.textDecoration = "underline";
+  descriptionContainer.appendChild(ingredientsTitle);
+
+  const ingredientsList = document.createElement("ul");
+  ingredientsList.classList.add("ingredientsList");
+
+  for (let i = 0; i < recipeObject.ingredients.length; i++) {
+    let listItem = document.createElement("li");
+    let listItemName = document.createElement("div");
+    let listItemAmount = document.createElement("div");
+    listItem.appendChild(listItemName);
+    listItem.appendChild(listItemAmount);
+    listItemName.innerText = recipeObject.ingredients[i].NAME;
+    listItemAmount.innerText = recipeObject.ingredients[i].AMOUNT;
+    ingredientsList.appendChild(listItem);
+  }
+  descriptionContainer.appendChild(ingredientsList);
+
+  const descriptionTitle = document.createElement("p");
+  descriptionTitle.classList.add("recipeDescription");
+  descriptionTitle.innerText = "Description:";
+  descriptionTitle.style.textDecoration = "underline";
+  const description = document.createElement("p");
+
+  description.innerText = recipeObject.description;
+  descriptionContainer.appendChild(descriptionTitle);
+  descriptionContainer.appendChild(description);
+}
+
+//**************** Add new recipe Button *****************/
+const addRecipeBtn = document.getElementById("addRecipeBtn");
+addRecipeBtn.addEventListener("mouseover", function (e) {
```

Comment:
It's great that you experimented with mouse events! But it is quite overkill in your situation. You could achieve the same result in CSS with 
```css
#addRecipeBtn:hover {
  /* the .btnMouseOver style */
}
```